### ISSUE TITLE
Unable to donate money to `tig` project

### ISSUE THREAD

**Issue by agalloch** [jonas/tig#1133]
Hello, tig team! 👋 

I've been using `tig` for several years now and I often come to "Wow, tig is awesome, I wanna give those guys a boost!" but am unable to find a donation page. The topic has been brought up previously here https://github.com/jonas/tig/issues/852 but it doesn't mention a donation page either.

Is there a way to support you guys? 🙂

**Comment by graves501** [comment]
>"Wow, tig is awesome, I wanna give those guys a boost!" 

I agree, there should be a way to support this project by donating some money :smile:

**Comment by avioli** [comment]
@jonas Can you setup a [Buy me a coffee](https://buymeacoffee.com/) page for yourself and add a link in here or in the README? [Wise send-money](https://wise.com/send-money/)... or [GitHub Sponsors](https://github.com/sponsors)... or whatever works for you.

I too have used `tig` for several years now and have recommended it to many peer.
